mysql-test/suite/tianmu/r/issue1352.result

```diff
@@ -0,0 +1,25 @@
+DROP DATABASE IF EXISTS issue1352_test;
+CREATE DATABASE issue1352_test;
+USE issue1352_test;
+CREATE TABLE test (dt TIMESTAMP) engine=tianmu;
+insert test values(now());
```

Comment:
adds some tests for deterministic-volatile function.
F.Y.I: https://stackoverflow.com/questions/70088382/deterministic-volatile-function-in-sql